hazelcast-jclouds/src/main/java/com/hazelcast/jclouds/ComputeServiceBuilder.java

```diff
@@ -218,17 +251,18 @@ public boolean apply(ComputeMetadata nodeMetadata) {
                     return false;
                 }
                 for (AbstractMap.SimpleImmutableEntry entry: tagPairs) {
-                    if (!nodeMetadata.getUserMetadata().get(entry.getKey()).
-                            equals(entry.getValue())) {
-                        return false;
+                    String value = nodeMetadata.getUserMetadata().get(entry.getKey());
+                    if (value != null && value.equals(entry.getValue())) {
```

Comment:
now we are make `OR` operation between tags. It was `AND` operation before...
which one is more usable? @mesutcelik